admin/tools/docker/deployment/kubernetes/orchestration/start.sh

```diff
@@ -77,7 +77,9 @@ host_log_dir: $HOST_LOG_DIR
 host_tmp_dir: $HOST_TMP_DIR
 host: $host
 image: $CONTAINER_IMAGE
+image_mariadb: qserv/mariadb_scisql:10.1.25
 master_hostname: $MASTER
+mysql_root_password: CHANGEME
```

Comment:
Should we also publish root password on LSST web page in case google does not find it here? 😏

Reply:
I have create a ticket to manage credentials more cleanly